java-workspace/Dockerfile

```diff
@@ -1,21 +1,25 @@
-FROM openjdk:8-jre
+FROM openjdk:8-jre-buster
 
-MAINTAINER Sebastian Mandrean <sebastian@urb-it.com>
+LABEL maintainer="pd@urbit.com"
```

Comment:
`MAINTAINER` is deprecated

Reply:
So... remove it?

Reply:
We could, but `LABEL` is the preferred way of setting a maintainer now. But since these are just internal images it doesn't make much of a difference to keep it. Since it's here already I'll leave it.